src/common-models/shift-info.model.ts

```diff
@@ -1,5 +1,47 @@
 import { ScheduleKey } from "../api/persistance-store.model";
 
+export interface Shift {
+  code: string;
+  name: string;
+  from: number;
+  to: number;
+  color: string;
+  isWorkingShift?: boolean;
+}
+
+export const shifts: { [id: string]: Shift } = {
+  R: { code: "R", name: "Rano", from: 7, to: 15, color: "FFE880", isWorkingShift: true },
+  P: { code: "P", name: "Popołudnie", from: 15, to: 19, color: "B3E3FF", isWorkingShift: true },
+  D: { code: "D", name: "Dzień", from: 7, to: 19, color: "CBEECB", isWorkingShift: true },
+  N: { code: "N", name: "Noc", from: 19, to: 7, color: "B7BCC7", isWorkingShift: true },
+  DN: { code: "DN", name: "Dzień + Noc", from: 7, to: 7, color: "", isWorkingShift: true },
+  PN: {
+    code: "PN",
+    name: "Popołudnie + Noc",
+    from: 19,
+    to: 7,
+    color: "",
```

Comment:
Myślę, że zamiast dawać puste stringi, lepiej by było zrobić kolor opcjonalną wartością, bo wtedy to będzie czytelniej

Reply:
Założyłam, że docelowo wszystkie zmiany będą miały przypisany jakiś kolor, który po prostu nie został jeszcze zdefiniowany na Figmie ;)